modules/sdk-core/src/bitgo/lightning/iLightning.ts

```diff
@@ -155,10 +149,11 @@ export type GetInvoicesResponse = t.TypeOf<typeof GetInvoicesResponse>;
 
 export interface ILightning {
   createInvoice(params?: CreateInvoiceParams): Promise<CreateInvoiceResponse>;
-  createDepositAddress(params?: CreateDepositAddressParams): Promise<CreateDepositAddressResponse>;
+  createDepositAddress(): Promise<CreateDepositAddressResponse>;
   payInvoice(params?: PayInvoiceParams): Promise<PayInvoiceResponse>;
   getBalance(): Promise<GetBalanceResponse>;
   withdraw(params?: LightningWithdrawalParams): Promise<WithdrawResponse>;
   deposit(params?: LightningDepositParams): Promise<DepositResponse>;
-  getInvoices(params?: GetInvoicesParams): Promise<GetInvoicesResponse>;
+  // Implemented in lnd-middleware but not in wallet-platform
```

Comment:
This comment isn't helpful for those who don't have context on the lightning implementation - please remove.